pandas/tests/extension/decimal/array.py

```diff
@@ -15,6 +15,17 @@ class DecimalDtype(ExtensionDtype):
     name = 'decimal'
     na_value = decimal.Decimal('NaN')
 
+    def __init__(self, context=None):
+        self.context = context or decimal.getcontext()
+
+    def __eq__(self, other):
```

Comment:
Our default __eq__ is dangerous for parametrized dtypes, depending on whether the parameters appear in the name :/